pkg/system/chtimes.go

```diff
@@ -2,14 +2,28 @@ package system
 
 import (
 	"os"
+	"syscall"
 	"time"
+	"unsafe"
 )
 
+// maxTime returns the maximum time that can be set by os.Chtimes on this platform
+// 64 bit Timespec supports up to 2262 and 32 bit Timespec supports up to 2038
+func maxTime() time.Time {
```

Comment:
Ah, that is how to do that. I was wasn't sure if ```init()``` was the correct place for that or not. Will do.